storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/PostgresStore.scala

```diff
@@ -0,0 +1,123 @@
+package com.twitter.storehaus.postgres
+
+import com.twitter.storehaus.Store
+import com.twitter.util.{Future, Time}
+import roc.postgresql.{Client, Element, Request, Result}
+
+/**
+  * Key value storage with PostgreSQL (>9.5) as backend
+  *
+  * Created by ponkin on 9/24/16.
+  */
+object PostgresStore {
+
+  def apply[K: Show : Read, V: Show : Read](client: Client, table: String, kCol: String, vCol: String) =
+    new PostgresStore[K, V](client, table, kCol, vCol)
+}
+
+class PostgresStore[K: Show : Read, V: Show : Read]
```

Comment:
would it be too much work to add a `PostgresMergeableStore` that does `merge` in a transaction? That would be nice to have for several use cases.